src/components/organisms/ValidationPane/CRDsSchemaValidation/styled.ts

```diff
@@ -0,0 +1,29 @@
+import {Input} from 'antd';
```

Comment:
Rename the styled file to `CRDsSchemaValidation.styled.tsx` so we follow the same convention as for the other files